gen_cert.py

```diff
@@ -229,7 +217,12 @@ def interstitial_factory():
             self.long_course = long_course or cert_data.get('LONG_COURSE', '').encode('utf-8')
             self.issued_date = issued_date or cert_data.get('ISSUED_DATE', '').encode('utf-8') or 'ROLLING'
             self.interstitial_texts = collections.defaultdict(interstitial_factory())
-            self.interstitial_texts.update(cert_data.get('interstitial', {}))
+            interstitial_dict = {k.encode('utf8'): v.encode('utf8') for k, v in cert_data.get('interstitial', {}).items()}
+            self.interstitial_texts.update(interstitial_dict)
+            self.locale = cert_data.get('LOCALE', settings.DEFAULT_LOCALE).encode('utf-8')
```

Comment:
It seems that you're supposed to encode when reading from the yaml, just in case there's UTF-8 stuff in there? I just copied what they were doing above...also, if you don't do this, it dies when trying to run the cert render.